.github/scripts/torchao_model_releases/quantize_and_upload.py

```diff
@@ -206,7 +206,7 @@ def _untie_weights_and_save_locally(model_id):
 
 _int4_quant_code = """
 from torchao.quantization import Int4WeightOnlyConfig
-quant_config = Int4WeightOnlyConfig(group_size=128, packing_format="tile_packed_to_4d", int4_choose_qparams_algorithm="hqq", version=2)
+quant_config = Int4WeightOnlyConfig(group_size=128, packing_format="tile_packed_to_4d", int4_choose_qparams_algorithm="hqq")
```

Comment:
just found we also need to update packing_format to `int4_packing_format` I have made change locally, can push these changes before land.